shuttle/templates/schedules/driver_logs.html

```diff
@@ -16,24 +15,32 @@ <h4>Driver Shuttle Logs</h4>
             </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td class="driver">*Insert Driver here*</td>
-                    <td class="location">*Insert location here*</td>
-                    <td class="check-in">*Insert time in here*</td>
-                    <td class="check-out">*Insert time out here*</td>
-                </tr>
-                <tr>
-                    <td class="driver">*Insert Driver here*</td>
-                    <td class="location">*Insert location here*</td>
-                    <td class="check-in">*Insert time in here*</td>
-                    <td class="check-out">*Insert time out here*</td>
-                </tr>
-                <tr>
-                    <td scope="driver">*Insert Driver here*</td>
-                    <td scope="location">*Insert location here*</td>
-                    <td class="check-in">*Insert time in here*</td>
-                    <td class="check-out">*Insert time out here*</td>
-                </tr>
+                {% for key,parent_dict_item in shuttle_logs.items() %}
+                    <tr>
+                        {% if parent_dict_item['location'] != None %}
+                            {% if parent_dict_item['name'] == None %}
+                                <td class="driver"></td>
+                            {% else %}
+                                <td class="driver">{{ parent_dict_item['name'] }}</td>
+                            {% endif %}
+                            {% if parent_dict_item['location'] == None %}
+                                <td class="location">N/A</td>
+                            {% else %}
+                                <td class="location">{{ parent_dict_item['location'] }}</td>
+                            {% endif %}
+                            {% if parent_dict_item['arrival_time'] == None %}
+                                <td class="check-in"></td>
+                            {% else %}
+                                <td class="check-in">{{ parent_dict_item['arrival_time'] }}</td>
```

Comment:
Put the if statement inside of the <td class="check-in"></td> so that only the data has to be inputted and not the html structure that already is there no matter what.

IE
Instead of:
```
{% if parent_dict_item['arrival_time'] == None %}
    <td class="check-in"></td>
{% else %}
    <td class="check-in">{{ parent_dict_item['arrival_time'] }}</td>
{% endif %}
```
Do this
```
<td class="check-in>
    {% if parent_dict_item['arrival_time'] != None %}
        {{ parent_dict_item['arrival_time'] }}
    {% endif %}
</td>
```

Do this in all the instances that it can be done.